controller/greet.js

```diff
@@ -0,0 +1,3 @@
+export const greet = (req, res) => {
+  res.send("Hello World");
```

Comment:
Learn about different http errorcode, and add http code in response

Reply:
Done